x/market/types/types.go

```diff
@@ -12,12 +12,12 @@ import (
 type OrderState uint8
 
 const (
+	// OrderClosed is used when state of order is closed
+	OrderClosed OrderState = iota // default (0)
 	// OrderOpen is used when state of order is open
-	OrderOpen OrderState = iota
+	OrderOpen
 	// OrderMatched is used when state of order is matched
-	OrderMatched OrderState = iota
-	// OrderClosed is used when state of order is closed
-	OrderClosed OrderState = iota
+	OrderMatched
```

Comment:
* `Closed` is not the initial state.
* Some linters will complain about this formatting.  Why are you changing it to begin with?

Reply:
the default value for `uint8` is 0, that means that all default states would have been open

Reply:
check https://github.com/uber-go/guide/blob/master/style.md#start-enums-at-one for reference

Reply:
> Some linters will complain about this formatting.

which linter complain about this format?

Reply:
Yes of course I know the initial value for `iota`.

When orders are created they are `open`, hence that being the default state.  `closed` is an end state.

I think it would be reasonable to not use `0` as a value at all to make sure things are always explicit.  To do this, replace `iota` with `iota + 1`